localstripe/resources.py

```diff
@@ -3852,7 +3858,10 @@ def _api_approve(cls, id: str, amount=None, metadata=None, **kwargs):
             logger.warning('Unexpected ' + ', '.join(kwargs.keys()))
             raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
 
-        amount = try_convert_to_int(amount)
+        if amount is not None:
+            amount = try_convert_to_int(amount)
+        else
+            amount = 0
```

Comment:
Also because we assert for amount being greater than 0 if it's present, this will throw an assertion exception.